_data/fr/general.yml

```diff
@@ -0,0 +1,13 @@
+footer:
+  copyright: Logiciel développé par <a href="https://github.com/corrados" target="_blank" rel="noreferrer">Volker Fischer</a> et les <a href="https://github.com/corrados/jamulus/graphs/contributors" target="_blank" rel="noreferrer">contributeur·trice·s</a>
+  alt:
+    github:          "Dépôt Jamulus sur GitHub"
+    facebook:        "Groupe Facebook Jamulus"
+    sourceforge:     "Optenez Jamulus, Internet Jam Session Software, sur SourceForge.net. Téléchargement rapide, sûr et gratuit de logiciels libres et à code source ouvert."
```

Comment:
"Optenez": mistake -> "Obtenez"
That said, a proper translation from the original English sentence would be: "Obtenir un logiciel de session de bœuf sur Internet à SourceForge.net"